src/main/java/mate/academy/spring/model/dto/request/UserRequestDto.java

```diff
@@ -0,0 +1,11 @@
+package mate.academy.spring.model.dto.request;
+
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+public class UserRequestDto {
+    private String email;
+    private String password;
+}
```

Comment:
same here

Reply:
I use it at register(), in authenticator controller method

Reply:
> I use it at register(), in authenticator controller method

Yeah, you are right, didn't noticed that